### DISCUSSION TITLE
Resource Pack Sounds

### DISCUSSION THREAD

**Discussion by DevBD1** [Zrips/CMI#4351] [Feature Requests]
We should be able to use custom (resource pack) sounds with /cmi sound command.

---
### Basic explanation as to why this feature is needed and/or useful to have: ###

Because we cannot use vanilla playsound command on the console for custom sounds if player is not in main world ( level-name in server.properties )

**Comment by CloudeLecaw** [comment]
This does not make sense. It woukd create errors for everyone or play nothing if you dont have the resources Pack since these Packs are Client based. Cmi just uses sounds that are server based in the server.jar

**Comment by EndlessLegends** [comment]
I seem to understand your opinion.
Is it correct that the following commands in the following packs are not registered in cmi?

resourcepack/minecraft/sounds.json >>>>>>>>>>

{
    "custom.cmi.test": {
        "replace": true,
        "sounds": [
            {
                "name": "custom/cmi/test",
                "stream": true
            }
        ]
    }

.ogg.... etc...

ingame >>>>>>>>>>
/minecraft:playsound minecraft:custom.cmi.test record zrips 90 1 1
NOW PLAYING!!!

/cmi sound cust..... ????????
* play sound at target location...

I'm also looking forward to hearing my songs customized in the resource pack with the cmi sound command. Your suggestion is great.

I got help from Google Translator.

**Comment by CloudeLecaw** [comment]
> Is it correct that the following commands in the following packs are not registered in cmi?

you got it, they are not registered by cmi normaly because they are not part of the server.jar
